src/System.Data.SqlClient/tests/Directory.Build.props

```diff
@@ -1,11 +0,0 @@
-<Project>
-
-  <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props, $(MSBuildThisFileDirectory)..))" />
-
-  <PropertyGroup>
-    <!-- Workaround: There are many .csprojs under this tests directory which do not meet
-         the criteria of IsTestProject but still meet the criteria of IsSourceProject. -->
-    <IsSourceProject>false</IsSourceProject>

```

Comment:
Why not keep this file and change this line for 
`    <IsTestUtilityProject>true</IsTestUtilityProject>`
instead of modifying all the csprojs in this folder?

Reply:
The `IsTestUtilityProject` property identifies test support projects that aren't invoked directly. So that property shouldn't be put on the real test projects, ie. System.Data.SqlClient.Tests.csproj.

Reply:
Ah, okay, got it.